src/Symfony/Component/Notifier/Bridge/SmsBiuras/Tests/SmsBiurasTransportFactoryTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\SmsBiuras\Tests;
+
+use Symfony\Component\Notifier\Bridge\SmsBiuras\SmsBiurasTransportFactory;
+use Symfony\Component\Notifier\Test\TransportFactoryTestCase;
+use Symfony\Component\Notifier\Transport\TransportFactoryInterface;
+
+final class SmsBiurasTransportFactoryTest extends TransportFactoryTestCase
+{
+    /**
+     * @return SmsBiurasTransportFactory
+     */
+    public function createFactory(): TransportFactoryInterface
+    {
+        return new SmsBiurasTransportFactory();
+    }
+
+    public function createProvider(): iterable
+    {
+        yield [
+            'smsbiuras://host.test?from=0611223344&test_mode=1',
+            'smsbiuras://uid:api_key@host.test?from=0611223344&test_mode=1',
+        ];
```

Comment:
please add a test case without test_mode parameter

and with a test_mode_parameter and `0` as value, thanks